server/apps/main/helpers.py

```diff
@@ -167,6 +202,21 @@ def make_tags(data):
     tags = [tag for tag in tags if bool(tag) == True]
     return tags
 
+def prepare_metadata(uuid, timestring, data):
+    # The following items will be stripped from the metadata dictionary
+    metadata_mask = [
+        "experience_text",
+        "difference_text",
+        "title_text"
```

Comment:
I think we should keep the title-text in the metadata, as the _my stories_ view at `/main/my_stories/` uses them for its table to display the title for all experiences of a user without having to download all files from OH.

Reply:
Whops, you're absolutely right! The `title_text` of the metadata isn't used, instead it uses the `description`, which we set to be the `title_text` in the `upload` function! I think in that case it's okay to leave it as is, as the `description` field is mandatory metadata when uploading to OH. 

Though, maybe we should truncate the title text to the first 100 characters when coping it into the `description` field, as [according to our documentation on OH](https://www.openhumans.org/direct-sharing/oauth2-data-upload/), the field only accepts that many characters?

Reply:
I have to admit I didn't know it either, I originally was about to recommend to just remove the `description` field but then thought that I should check the OH docs before doing so and that's when I saw it 😂